src/Components/Authentication/Desktop/SignUpForm.tsx

```diff
@@ -39,9 +39,11 @@ export const SignUpForm: FormComponentType = props => {
         handleBlur,
         handleSubmit,
         isSubmitting,
-        isValid,
+        dirty,
         status,
```

Comment:
what is `dirty`?

Reply:
`dirty` is set to `false` if none of the fields in the form have been touched (the value has been changed), `true` if one of those has changed.